library/systemd/test/yast2/system_service_test.rb

```diff
@@ -129,18 +121,6 @@
       expect(system_services).to be_all(Yast2::SystemService)
       expect(system_services.map(&:service)).to eq([apparmor, cups])
     end
-
-    context "when some service is not found" do
```

Comment:
I would add a test to reflect to happens when a service is not found.

Reply:
Although this class it is not responsible for that anymore, you are right that we should cover that case somehow. So I've added an additional test to `SystemdService` which covers that case.